src/AutosuggestContainer.js

```diff
@@ -69,13 +69,14 @@ export default class AutosuggestContainer extends Component {
     renderSectionTitle: PropTypes.func,
     getSectionSuggestions: PropTypes.func,
     focusInputOnSuggestionClick: PropTypes.bool,
+    selectFirstSuggestion: PropTypes.bool,
     theme: PropTypes.object,
     id: PropTypes.string
   };
 
   static defaultProps = {
     onSuggestionsUpdateRequested: noop,
-    shouldRenderSuggestions: value => value.trim().length > 0,
+    shouldRenderSuggestions: value => value && value.trim().length > 0,
```

Comment:
Default props in the library? Where is that?
If the consumer doesn't set `value` to a string, I'd prefer to throw an error in this case. `value` should always be a string.